### ISSUE TITLE
Feature Request: Import Multiple CSVs

### ISSUE THREAD

**Issue by daniel-samson** [salesagility/SuiteCRM#1176]
In relation  to [SuiteCRM Forum Post](https://suitecrm.com/forum/suitecrm-7-0-discussion/8712-how-to-upload-contacts-and-accounts-merged-together-in-cvs-file#30330)

User would like to be able to uploaded multiple CSVs at the same time for different modules. This should resolve records which are related.

**Comment by vivasergei** [comment]
Hi

One of the first ways to implement could be to open an account in crm and link to this account multiple contact id's.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Feature Request: Import Multiple CSVs](https://trello.com/c/TffL18bU/110-feature-request-import-multiple-csvs)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
